### ISSUE TITLE
PAR::Dist::blib_to_par quotes the version number of the dist when making the PAR file

### ISSUE THREAD

**Issue by tehmoth** [rschupp/PAR-Dist#2]
For example, making a PAR out of `.PAR::Dist`
```
$ perl -MPAR::Dist -e'blib_to_par'
Successfully created binary distribution 'PAR-Dist-'0.52'-x86_64-linux-gnu-thread-multi-5.32.1.par'.
Its contents are accessible in compliant browsers as:
    jar:file:///home/perly/.cpanm/work/1718679653.31043/PAR-Dist-0.52/PAR-Dist-'0.52'-x86_64-linux-gnu-thread-multi-5.32.1.par!/MANIFEST
```
`PAR-Dist-'0.52'-x86_64-linux-gnu-thread-multi-5.32.1.par` instead of `PAR-Dist-0.52-x86_64-linux-gnu-thread-multi-5.32.1.par`, using the version from META / MYMETA.    When the version number in META is a number, not a string (despite CPAN::Meta::Spec), the file name is generated correctly.   Also if the version is explicitly set in the call to `blib_to_par`, the filename is generated correctly

```
 perl -MPAR::Dist -e'blib_to_par( version => "0.52" )'
Successfully created binary distribution 'PAR-Dist-0.52-x86_64-linux-gnu-thread-multi-5.32.1.par'.
Its contents are accessible in compliant browsers as:
    jar:file:///home/perly/.cpanm/work/1718679653.31043/PAR-Dist-0.52/PAR-Dist-0.52-x86_64-linux-gnu-thread-multi-5.32.1.par!/MANIFEST
```
the lines 
```
           elsif (/^version:\s+(.*)/) {
                $version ||= $1;
            }
        }
```
and
```
          elsif (/^VERSION\s+=\s+(.*)$/) {
                $version ||= $1;
            }
```

**Comment by tehmoth** [comment]
`ExtUtils::MM_Unix::get_version` seems to use `eval` for this for the  `package .. VERSION ...` case, but for the META / MYMETA case a YAML module could be used to parse it instead

**Comment by rschupp** [comment]
@tehmoth Thanks, fixed in 0.53
